apps/web/src/pages/get-started/layout/GetStartedTab.tsx

```diff
@@ -1,23 +1,65 @@
 import styled from '@emotion/styled';
 import { Grid } from '@mantine/core';
 
-import { colors, Text } from '@novu/design-system';
+import { ArrowLeft, Button, colors, Text } from '@novu/design-system';
+import { useMemo } from 'react';
 
 import Card from '../../../components/layout/components/Card';
 import { Timeline } from '../components/timeline/Timeline';
-import { OnboardingUseCase } from '../consts/types';
 import { AdditionInformationLink } from '../components/AdditionInformationLink';
+import { IOnboardingUseCaseViewContext, OnboardingUseCase } from '../consts/types';
 
-type IGetStartedTabProps = OnboardingUseCase;
+export interface IGetStartedTabProps extends OnboardingUseCase, IOnboardingUseCaseViewContext {}
+
+const StyledTimeline = styled(Timeline)<{ hasBottomSection?: boolean }>(
+  ({ hasBottomSection }) => `
+  margin-bottom: ${hasBottomSection ? '1.5rem' : 0};
+`
+);
+
+const TabBreadcrumb = styled(Button)(
+  ({ theme }) => `
+  padding: 0;
+  background: none;
+  border: none;
+  color: ${theme.colors.gray[7]};
+  margin-bottom: 1rem;
+  height: inherit;
+  
+  display: flex;
+  
+  & span {
+    background-image: none;
+    font-weight: normal;
+  }
+  `
+);
+
+export function GetStartedTab({ setView, currentView, views, ...tabProps }: IGetStartedTabProps) {
+  const shouldShowBreadcrumb = !!currentView && views && !!views[currentView];
+
+  const { steps, Demo, title, description, BottomSection, type, useCaseLink } = useMemo(() => {
+    if (!currentView || !views) {
+      return tabProps;
+    }
+
+    return views[currentView] ?? tabProps;
+  }, [views, currentView, tabProps]);
 
-export function GetStartedTab({ steps, Demo, title, description, useCaseLink, type }: IGetStartedTabProps) {
   return (
     <Grid align="stretch" justify={'space-between'}>
       <Grid.Col span={3} mt={12}>
```

Comment:
I would also suggest setting this column a min-width because on the smallest screen that we support it has a huge gap IMO, 18.75rem looks fine. But when shrinking the viewport width the animation will jump below, having nowrap on parent helps but creates the scroll :/

![Screenshot 2024-02-21 at 19 14 02](https://github.com/novuhq/novu/assets/2607232/8a4c7423-e2ad-41de-850c-4b64ca37ea49)
![Screenshot 2024-02-21 at 19 20 49](https://github.com/novuhq/novu/assets/2607232/18f0c1f9-4307-4a64-a95b-9ec202437e80)